src/main/java/core/basesyntax/service/RegistrationServiceException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class RegistrationServiceException extends RuntimeException {

```

Comment:
```suggestion
public class RegistrationException extends RuntimeException {
```

Exception name should be more abstract